src/Sulu/Bundle/AdminBundle/Resources/public/js/components/login/main.js

```diff
@@ -161,190 +177,174 @@ define([], function() {
          *
          * @event sulu.login.[INSTANCE_NAME].initialized
          */
-        INITIALIZED = function() {
+        INITIALIZED = function () {
             return createEventName.call(this, 'initialized');
         },
 
-        createEventName = function(postfix) {
+        createEventName = function (postfix) {
             return 'sulu.login.' + ((!!this.options.instanceName) ? this.options.instanceName + '.' : '') + postfix;
         };
 
     return {
-
         /**
-         * Initializes the component
+         * Initialize component
          */
-        initialize: function() {
+        initialize: function () {
```

Comment:
@nnatter no spaces before round brackets. If this is due to an IDE-config, please change it